### PR TITLE
Fix design matrice creation

### PR DESCRIPTION
**Pull request by LaurenceKuhl**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/crisprseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/crisprseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by LaurenceKuhl** [comment]
hi @mirpedrol sorry for the double quoting :D 
We fixed the error and it seemed to be coming from the nextflow edge version + some channel handling in the edition workflow. We found a workaround but could you please double check this doesn't change anything fundamental in the workflow? Thank you:)

File: workflows/crisprseq_targeted.nf [hunk 1]
```diff
@@ -578,16 +578,6 @@ workflow CRISPRSEQ_TARGETED {
         true
     )
     .bam
-    .map {
```
Comment (mirpedrol) [inline workflows/crisprseq_targeted.nf:581]: I added this because I didn't want to use the `meta.id`, which will be the name of the sample, but the name of the file which will be the name of the reference (or any other name the user gave to this file). When adding `template-align` to the name, if I resume the pipeline, it keeps adding `template-align`, that's why I wrote this workaround with map.

Was this bit causing the error? 

I think your fix will work good (I haven't tested it), and I agree using `meta.id` is easier and cleaner, and users don't need to inspect this file so it is not a big deal if we rename them. 
TL;DR: I agree with the change 😄

**Review by mirpedrol — APPROVED** [review]

File: workflows/crisprseq_targeted.nf (hunk 1 above)
Comment (mirpedrol) [reply workflows/crisprseq_targeted.nf:581]: No I think your fix will work fine, thanks :)
